module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,65 @@
+#include "compression.hpp"
+
+#include <algorithm>
+#include <array>
+#include <iostream>
+#include <utility>
+#include <vector>
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(std::array<std::array<uint8_t, width>, height>& bitmap) {
+    std::vector<std::pair<uint8_t, uint8_t>> compressedData;
+    compressedData.reserve(width * height);
+
+    for (const auto& row : bitmap) {
+        if (row.empty())
```

Comment:
This is array :) size is defined at compilation time, so can't be empty ;)

Reply:
Removed this